x/concentrated-liquidity/lp_test.go

```diff
@@ -605,20 +691,15 @@ func (s *KeeperTestSuite) TestUpdatePosition() {
 			expectedPoolLiquidity: sdk.ZeroDec(),
 			expectedError:         true,
 		},
-		"try updating with ticks outside existing position's tick range": {
-			poolId:                    1,
-			ownerIndex:                0,
-			lowerTick:                 DefaultUpperTick + 1,
-			upperTick:                 DefaultUpperTick + 100,
-			liquidityDelta:            DefaultLiquidityAmt,
-			amount0Expected:           sdk.NewInt(18395),
-			amount1Expected:           sdk.ZeroInt(),
-			expectedTickLiquidity:     DefaultLiquidityAmt,
-			expectedPositionLiquidity: DefaultLiquidityAmt,
-			// only liquidity that has been added from create position.
-			// no liquidity has been added by updating position since tick was outside range.
-			expectedPoolLiquidity: DefaultLiquidityAmt,
-			expectedError:         false,
+		"try updating with ticks outside existing position's tick range - error because fee accumulator is uninitialized": {
+			poolId:          1,
+			ownerIndex:      0,
+			lowerTick:       DefaultUpperTick + 1,
+			upperTick:       DefaultUpperTick + 100,
+			liquidityDelta:  DefaultLiquidityAmt,
+			amount0Expected: sdk.NewInt(18395),
+			amount1Expected: sdk.ZeroInt(),
```

Comment:
amount0 and amount1 expected should be nil, we just don't check them when expecting error

Reply:
addressed